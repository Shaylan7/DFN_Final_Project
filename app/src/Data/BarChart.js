import * as React from "react";

import { Bar } from "react-chartjs-2";

const BarChart = () => {
  return (
    <div>
      hello
      <Bar
        data={{
          labels: ["Consumer Packaged Goods", "Food"],
          datasets: [
            {
              label: "# of Votes",
              data: [12, 19, 3, 5, 2, 3],
              backgroundColor: [
                "rgba(255, 99, 132, 0.2)",
                "rgba(54, 162, 235, 0.2)",
                "rgba(255, 206, 86, 0.2)",
                "rgba(75, 192, 192, 0.2)",
                "rgba(153, 102, 255, 0.2)",
                "rgba(255, 159, 64, 0.2)",
              ],
            },
          ],
        }}
        height={400}
        width={600}
      />
    </div>
  );
};

export default BarChart;
