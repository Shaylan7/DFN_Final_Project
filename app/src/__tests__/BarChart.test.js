import React from 'react';
import BarChart from '../Data/BarChart';
import { create } from 'react-test-renderer'; 

describe('My first snapshot test',()=>{
    test('testing BarChart component', () => {
    let tree = create(<BarChart />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})