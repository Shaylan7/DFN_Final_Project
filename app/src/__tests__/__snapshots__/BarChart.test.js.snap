// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first snapshot test testing BarChart component 1`] = `
<div
  class="chartbox"
>
  <canvas
    height={10}
    onClick={[Function]}
    role="img"
    width={50}
  />
</div>
`;
